src/olympia/amo/views.py

```diff
@@ -90,6 +90,15 @@ def handler404(request, exception=None, **kwargs):
 
 @non_atomic_requests
 def handler500(request, **kwargs):
+    if getattr(request, 'is_api', False):
+        # API exceptions happening in DRF code would be handled with by our
+        # custom_exception_handler function in olympia.api.exceptions, but in
+        # the rare case where the exception is caused by a middleware or django
+        # itself, it might not, so we need to handle it here.
+        return HttpResponse(
+            json.dumps({'detail': 'Internal Server Error'}),
+            content_type='application/json',
+            status=500)
```

Comment:
if using the entire code from `custom_exception_handler` isn't practical, can we can have:
```
if settings.DEBUG:
            data['traceback'] = traceback.format_exc()
```
included?

Reply:
I will re-use same function